### PR TITLE
Remove call to deprecated UltiSnips#FileTypeChanged()

### LINKED ISSUE #7: Getting an error about UltiSnips
I recently updated django-plus in my vim instance and now I'm getting the following error. I currently have the latest version of UltiSnips and django-plus installed.

> Error detected while processing /home/chadgh/.dotfiles/vim/vim/bundle/django-plus.vim/after/ftplugin/python.vim:
line   15:
E117: Unknown function: UltiSnips#FileTypeChanged

### PR DESCRIPTION
**Pull request by eyemyth**
Fixes #7.

### CODE REVIEW

File: after/ftplugin/python.vim [hunk 1]
```diff
@@ -11,9 +11,6 @@ setlocal omnifunc=djangoplus#complete
 
 
 if exists(':UltiSnipsAddFiletypes')
-  " Ensure UltiSnipsEdit opens the htmldjango.snippets file
-  call UltiSnips#FileTypeChanged()
-
   " Add HTML snippets
```
Comment (blueyed) [inline after/ftplugin/python.vim]: Here the comment seems to be wrong: s/HTML/Django/

Comment (eyemyth) [inline after/ftplugin/python.vim]: Heh, didn't even notice that. I went ahead and fixed that too.

Comment (tweekmonster) [inline after/ftplugin/python.vim]: Ah yes, from me copying and pasting things.  😤

**Comment by tweekmonster** [comment]
Thanks!

**Pull request merged**